question-service/controller/question-controller.js

```diff
@@ -1,23 +1,18 @@
 
-const QuestionRepository = require('../repository/questions')
-const CategoryRepository = require('../repository/categories');
+const QuestionRepository = require('../repository/question-repository')
+const CategoryRepository = require('../repository/category-repository');
 const md = require('markdown-it')();
 
 async function createQuestion(req, res) {
     try {
         const { name, type, content, difficulty } = req.body;
         if (name && content && difficulty) {
             difficulty.toLowerCase()
-            await QuestionRepository.createTable();
-            await CategoryRepository.createTable();
-            console.log("Table created/initialized successfully")
-            QuestionRepository.add(name, content).then(id => { 
-                console.log("Question inserted succesfully with question id: " + id.q_id)
-                CategoryRepository.add(id.q_id, difficulty, [type]);
-                console.log("Category inserted succesfully")
-                return res.status(201).json({message: `Create new question successfully!`});
-            })
-
+            const question = await QuestionRepository.create(name, content);
+            console.log("Question created succesfully with id: " + question.id)
```

Comment:
Nice logging all around, we can consider moving to a different logging library in the future for production, perhaps we could standardize how we plan to do logging for the future.